### PR TITLE
Adding GitHub rules for SUSI

### PR DESCRIPTION
**Pull request by jig08**
.. wrt #769

### CODE REVIEW

**Comment by daminisatya** [comment]
@jig08 Is this aiming at answering only about GitHub profile details? 
I'm wondering what more it can answer! 
Also, why there is a filechange which is not relevant to the issue?

File: src/org/loklak/api/search/LocationWiseTimeService.java [hunk 1]
```diff
@@ -85,8 +85,7 @@ public static SusiThought locationWiseTime(String query) {
 				JSONObject obj = new JSONObject();
 				String l = e.getElementsByTag("a").text();
 				obj.put("location", l);
-				e = e.siblingElements().get(0);
-				String t = e.text();
+				String t = e.nextElementSibling().text();
```
Comment (sudheesh001) [inline src/org/loklak/api/search/LocationWiseTimeService.java]: Why is this line here ?

File: conf/susi/susi_cognition_000.json [hunk 2]
```diff
@@ -712,6 +712,16 @@
 			"actions":[	{"type":"answer", "select":"random", "phrases":[
 							"I like to chat with people!"
 						]}]
-		}
+		},
+		{
+			"keys"   :["github"],
+			"score"  :3000,
+			"example": "Susi, View the Github profile of mariobehling",
+			"phrases":[ {"type":"pattern", "expression":"* github * of *"}],
```
Comment (sudheesh001) [inline conf/susi/susi_cognition_000.json]: Any specific reason `* github * of *` ?
As far as I know the possible queries can be
- Show me the github profile of mariobehling
- What are the github profile stats of mariobehling
- Read mariobehling github profile 
- Give me more information from github about mariobehling
- Can you find mariobehling on github and give me information

File: src/org/loklak/api/search/LocationWiseTimeService.java (hunk 1 above)
Comment (jig08) [inline src/org/loklak/api/search/LocationWiseTimeService.java]: Oops. It is the commit of PR https://github.com/loklak/loklak_server/pull/768 added here mistakenly.

File: conf/susi/susi_cognition_000.json (hunk 2 above)
Comment (jig08) [inline conf/susi/susi_cognition_000.json]: Yes @sudheesh001 . This was just a specific pattern, we can have more too. Shall add 'em up.

Comment (sudheesh001) [inline conf/susi/susi_cognition_000.json]: Please add more patterns here which help make more sense off the githubscraper we have. :)

Comment (jig08) [inline conf/susi/susi_cognition_000.json]: Yes , doing it.

**Comment by jig08** [comment]
Follow up #773 for more.
